### ISSUE TITLE
Tooltip does not invoke on child element (BS5)

### ISSUE THREAD

**Issue by jagaxa** [twbs/bootstrap#31324]
Tooltip opens when I hover on the button but closes after my hovering reaches the inside element which is an svg icon. This seems like a non-issue in Bootstrap version 4 ([jsfiddle](https://jsfiddle.net/43tx2Lfs/)). 

Tooltip behavior in BS v5.0.0-alpha1:
![y75F3iHQ1C](https://user-images.githubusercontent.com/66206306/87857407-ac026600-c926-11ea-9f27-5a76821052ea.gif)

Sample code:
```
<button type="button" class="btn btn-primary" title="Preview" data-toggle="tooltip">
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="#FFF" fill="none">
        <polyline points="3 6 5 6 21 6"></polyline>
        <path d="M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2"></path>
        <line x1="10" y1="11" x2="10" y2="17"></line>
        <line x1="14" y1="11" x2="14" y2="17"></line>
    </svg>
</button>
```

**Comment by isteiger** [comment]
this was fixed in pull request #30928 and will be shipped in Alpha 2

**Comment by onassar** [comment]
I ran into this in a context where I couldn't upgrade to a newer version. The following helped:

``` css
[data-bs-toggle]::before {
    pointer-events: none;
}
div.tooltip {
    pointer-events: none;
}
```
